src/cmd/project_init.js

```diff
@@ -65,49 +74,87 @@ export class ProjectInitCommandSite extends CommandSite {
 
 	}
 
+	error(error) {
+		throw error;
+}
 }
 
 /**
  * Implements the project initialization command.
  */
 export class ProjectInitCommand extends Command {
 
+	static templateFile(name) {
+		return path.join(__dirname, 'templates', 'project', name);
+	}
+
+	static expandTemplate(fs, templateName, data) {
+		const readFile = promisify(fs.readFile);
+		return readFile(ProjectInitCommand.templateFile(templateName), 'utf-8')
+			.then(content => {
+				return require('underscore').template(content)(data);
```

Comment:
Could you break `require('underscore')` to its own method named something like `templateEngine` or something to make the dependency more visible.